### ISSUE TITLE
export / texture options

### ISSUE THREAD

**Issue by Mainman002** [jayanam/batex#2]
I think that you already have plans to add a drop down list for export options so if you do I was thinking about godot / gltf export options that would be useful. having a checkbox for Meshes->Materials would be good as this tends to pack all of the textures into each gltf model file (not very good for hdd space / modular assets) but still having the option to copy textures to a selected path or not might be useful. 

*export drop down list (you might be able to tie into the export list already in Blender but the options would need to change to match your addon features)

*gltf Meshes->Materials checkbox

*(if possible) Texture copy checkbox / path option

**Comment by Mainman002** [comment]
I know that I'm probably thinking too far into the future but maybe some day we'll have something like this ( I experimented with some programming and pretty much ended up breaking things, at least it's a nice concept gui xD ) 

![Screenshot_2019-08-14_01-35-55](https://user-images.githubusercontent.com/11281480/62999444-1156f880-be34-11e9-8056-00521e21e86b.png)

**Comment by jayanam** [comment]
That's not too far, have something like this on my planned issues:-) Is
ExportList the export format iek FBX, obj.... ?

**Comment by Mainman002** [comment]
yes .fbx, .dae. glb, etc I guess it could use a better name to describe it but idk I just made it up real quick as a test xD

**Comment by Mainman002** [comment]
I made some other changes while I was at it so I could use this in Unity as well as UE4 :)

![Screenshot_2019-08-14_02-49-58](https://user-images.githubusercontent.com/11281480/63003977-fa69d380-be3e-11e9-966a-050d933659dc.png)

**Comment by Mainman002** [comment]
![Screenshot_2019-08-14_02-51-56](https://user-images.githubusercontent.com/11281480/63004022-140b1b00-be3f-11e9-8edb-95a526dcaf22.png)

![Screenshot_2019-08-14_02-52-16](https://user-images.githubusercontent.com/11281480/63004026-179ea200-be3f-11e9-8bf1-606ccb42b831.png)

![Screenshot_2019-08-14_02-52-55](https://user-images.githubusercontent.com/11281480/63004029-1a999280-be3f-11e9-968e-46d90b6276e8.png)

**Comment by Mainman002** [comment]
on a personal note I'm really enjoying this addon, ugh It was gonna take me a long time to export my modular dungeon asset pieces. . . all 104+ models X_x your addon exported them all in less then a minute!

**Comment by jayanam** [comment]
Alright, I will add the bake_space_transform as well

**Comment by Mainman002** [comment]
I hope that I didn't overstep my boundaries or anything, I was just playin around and managed to add features that I used a lot in Unity3D. . . Yeah my first issues above were "what if" features that I'd like but at least the last few stuffs I implemented myself so hopefully that was alright with you? (cause I might add more features later Lol)

**Comment by jayanam** [comment]
@Mainman002 : Yes, it's alright, you can add a Pull Request as well so I can review and add the changes you made

**Comment by Mainman002** [comment]
Oo epic! alright I'll do some more testing and see if there's anything that might cause export errors and what not then make a pull request :)

**Comment by Mainman002** [comment]
awe man I'm sorry I tried to open a pull request but I couldn't figure out how that worked so I ended up just branching it. . . here's a link if it helps, I know that it's not ideal but meh -> https://github.com/Mainman002/batex

**Comment by jayanam** [comment]
Äh no man doesnt help... either dive into the topic and fully understand
the pull request or send me your changes and I will add them manually...

**Comment by jayanam** [comment]
Ok, I added apply transform, but will have to think about the export scale and also about the design when supporting several export formats (for these will have different options)

**Comment by Mainman002** [comment]
I don't think that it's so much that I don't know how to use pull requests, but so much that I just can't make them as you have to give members the authority to do so (if I read that correctly?) it doesn't seem to let me select my branch or any other branches of this repo so I think that's all up to you man. . . 

![Screenshot_2019-08-14_16-16-15](https://user-images.githubusercontent.com/11281480/63057014-e8c01480-beae-11e9-84f9-a18d1bf64cb3.png)

![Screenshot_2019-08-14_16-12-39](https://user-images.githubusercontent.com/11281480/63057105-12793b80-beaf-11e9-8b5b-09dd30de477e.png)

**Comment by jayanam** [comment]
Ok I see:-) But anyway I have to think it over. I added the apply transform
now but Im not sure about the scaling and other properties, because they
could interfere with the export format.
